single-checkbox-plugin/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@sap-devx/inquirer-gui-single-checkbox-plugin",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
it is not enough to update package.json, please run npm install locally and submit updated package-lock.json as a part of this PR